families/seth/rpc/src/calls/transaction.rs

```diff
@@ -67,11 +67,12 @@ pub fn send_transaction<T>(params: Params, mut client: ValidatorClient<T>) -> Re
         Error::invalid_params("`data` not set")))?;
 
     // Optional Arguments
+    let txn_count = client.get_account(from.clone(), BlockKey::Latest).map(|a| a.unwrap().nonce).unwrap();
```

Comment:
This seems unwieldy, but I don't know what the right Rust idiom should be here?

Reply:
The unwrap() will cause panics.  What I've done in similar expressions in other code is make sure that the last thing is a Result and then follow it with ? (shorthand for try!) which will cause the error to be returned or the value safely unwrapped.